src/jobsautoreport/report.py

```diff
@@ -83,12 +93,16 @@ def _get_top_n_jobs(
         computation_func: Callable[[str, list[JobDetails]], Any],
         n: int,
         is_reverse: bool = False,
-    ):
+    ) -> list[tuple[str, Any]]:
         distinct_jobs = {job.name for job in jobs}
         res = [
             (job_name, computation_func(job_name, jobs)) for job_name in distinct_jobs
         ]
-        res = sorted(res, key=lambda job: (job[1], job[0]), reverse=is_reverse)
+        res = (
+            sorted(res, key=lambda job: (job[1][2], job[0]), reverse=is_reverse)
+            if is_reverse
+            else sorted(res, key=lambda job: (job[1], job[0]), reverse=is_reverse)
```

Comment:
why do you need two implementations based on ``is_reverse``?
I don't understand what's going on here